lemur/users/service.py

```diff
@@ -26,6 +29,10 @@ def create(username, password, email, active, profile_picture, roles):
     :param roles:
     :return:
     """
+    strict_role_enforcement = current_app.config.get("LEMUR_STRICT_ROLE_ENFORCEMENT", False)
+    if strict_role_enforcement and not any(role.name in DEFAULT_ROLES for role in roles):
+        return dict(message="Default role required: admin, operator, read-only"), 400
```

Comment:
nit: may be worth rewording to clarify that at least one are required (not necessarily all)